pkg/sink/sink.go

```diff
@@ -57,6 +57,7 @@ type Sink struct {
 	Logger                 *zap.SugaredLogger
 	Recorder               *Recorder
 	Auth                   AuthOverride
+	WaitGroup              *sync.WaitGroup
```

Comment:
We should probably add a comment here to describe what this wait group is used for.

Reply:
done!